src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonEditorView.ts

```diff
@@ -95,7 +104,7 @@ export class ExecutionPlanComparisonEditorView {
 	private _bottomSimilarNode: Map<string, azdata.executionPlan.ExecutionGraphComparisonResult> = new Map();
 	private _latestRequestUuid: string;
 
-	private get _activeBottomPlanDiagram(): AzdataGraphView {
+	public get activeBottomPlanDiagram(): AzdataGraphView {
```

Comment:
The return type should be `AzdataGraphView | undefined` here (same for the activeTopPlanDiagram)

And make sure you're handling undefined properly in all cases - I see some code below where that isn't being checked.